syntax = "proto3";
package protos.playbooks;

import "google/protobuf/wrappers.proto";
import "protos/base.proto";


message ZenDutyIncidentEntryPoint {
  google.protobuf.StringValue service_name = 1;
  google.protobuf.StringValue incident_title = 2;
  google.protobuf.StringValue service_id = 3;
  google.protobuf.StringValue incident_id = 4;
}
